code/obj/item/misc_weapons.dm

```diff
@@ -1927,3 +1927,159 @@ obj/item/whetstone
 	New()
 		..()
 		src.setItemSpecial(/datum/item_special/rangestab)
+
+#define OFFENSIVE_MODE 1
+#define DEFENSIVE_MODE 2
+
+/obj/item/armblade
+	name = "Visicar Armblade"
+	desc = "An experimental mining device modified by the syndicate for their infiltrators."
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "arm_blade"
+	item_state = "arm_blade-D"
+	var/base_state = "arm_blade"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	hit_type = DAMAGE_BLUNT
+	flags = FPRINT | TABLEPASS
+	c_flags = EQUIPPED_WHILE_HELD
+	object_flags = NO_ARM_ATTACH
+	force = 5
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 5
+	stamina_damage = 25
+	stamina_cost = 5
+	stamina_crit_chance = 10
+	is_syndicate = TRUE
+	contraband = 6
+	w_class = W_CLASS_NORMAL
+	hitsound = 'sound/impact_sounds/Generic_Hit_1.ogg'
+	tool_flags = TOOL_CUTTING
+	attack_verbs = "slashes"
+	can_disarm = FALSE
+	uses_multiple_icon_states = 1
+	duration_remove = 6 SECONDS
+
+	var/mode = OFFENSIVE_MODE
+	var/attached = FALSE
+	var/obj/item/mining_tool/tool = null
+	var/image/shield = null
+
+	abilities = list(/obj/ability_button/armblade_toggle)
+
+	New()
+		..()
+		var/obj/item/mining_tool/T = new /obj/item/mining_tool(src)
+		src.tool = T
+		T.name = src.name
+		T.desc = src.desc
+		T.dig_strength = 5
+		T.hitsound_charged = 'sound/weapons/cutter.ogg'
```

Comment:
```suggestion
		T.hitsound_charged = 'sound/weapons/cutter.ogg'
```
I don't want this sound being associated with a traitor weapon, as I made it specifically for the plasma cutter mining tool.

Reply:
sure can put something together but im no sound guy

Reply:
if you can't come up with something suitable that's fine, this minor won't prevent the PR from being added to the game (there's far more important stuff to worry about). I can always make a sound for it assuming this PR passes other Dev's requirements.